charts/posthog/values.yaml

```diff
@@ -446,12 +446,14 @@ kafka:
     # -- Enable persistence using PVC
     enabled: true
     # -- PVC Storage Request for kafka volume
-    size: 20Gi
+    size: 80Gi
   # -- A size-based retention policy for logs
   # -- Should be less than kafka.persistence.size, minimum 1GB
-  logRetentionBytes: _15_000_000_000
+  logRetentionBytes: _2_333_333_333
```

Comment:
Add a comment explaining to users how they should calculate it if they change the size or partitions